src/sentry/static/sentry/app/views/routeError.jsx

```diff
@@ -77,7 +76,7 @@ class RouteError extends React.Component {
     if (this._timeout) {
       window.clearTimeout(this._timeout);
     }
-    $('.sentry-error-embed-wrapper').remove();
+    document.querySelector('.sentry-error-embed-wrapper').remove();
```

Comment:
Do we need to check that it exists before calling `remove()`?

Reply:
👍 good call. Let me test this out and see if I can just get away with `?.`

Reply:
nice I can